argo_jupyter_scheduler/utils.py

```diff
@@ -203,11 +209,16 @@ def gen_papermill_command_input(
     logger.info(f"output_path: {output_path}")
     logger.info(f"log_path: {log_path}")
     logger.info(f"html_path: {html_path}")
+    logger.info(f"papermill_status_path: {papermill_status_path}")
 
-    papermill = f"papermill -k {kernel_name} {input_path} {output_path}"
+    papermill = (
+        f"( papermill -k {kernel_name} {input_path} {output_path} ; "
```

Comment:
Since I am unable to test locally - what happens if notebook name includes spaces or parentheses? Should it be quoted?